perma_web/perma/templates/docs/developer/index.html

```diff
@@ -71,37 +71,18 @@ <h3 id="examples-in-curl" class="body-bh">Examples in cURL</h3>
     <div class="col-sm-8 docs reading-body">
     <h2 class="body-ah">Public Archives</h2>
 
-    <p class="body-text">(Note: these API routes are temporarily unavailable.)</p>
-
-    <p class="body-text">The Public Archives API endpoint helps you find all publicly available Perma.cc archives. You do not need an API Key to use this portion of the API.</p>
+    <p class="body-text">The Public Archives API endpoint helps you find publicly available Perma.cc archives. You do not need an API Key to use this portion of the API.</p>
     <p class="body-text">The base endpoint for this API is <code>{{base_url}}/public/archives/</code></p>
 
+    <p class="body-text">Only recently-made Perma Links are listed.</p>
+
+    <p>To inquire about a more complete dataset, please <a href="{% url 'contact' %}?subject=Public%20Perma%20Link%20Dataset%20Inquiry">please drop us a note</a>.</p>
```

Comment:
This language might be too subtle? If I were a user reading these docs, I might not parse this as "this API has a hard limit on the offset parameter."

I might suggest an edit like

```suggestion
    <p class="body-text">This API provides access to the most recently created Perma Links. To inquire about access to data beyond this limit, <a href="{% url 'contact' %}?subject=Public%20Perma%20Link%20Dataset%20Inquiry">please drop us a note</a>.</p>
```

Reply:
Thanks, I like that suggestion a lot.